src/react/actions/__tests__/s3object.test.jsx

```diff
@@ -75,6 +82,21 @@ describe('s3object actions', () => {
             fn: actions.toggleAllObjects(true),
             expectedActions: [dispatchAction.TOGGLE_ALL_OBJECTS_ACTION(true)],
         },
+        {
+            it: 'should return RESET_OBJECT_METADATA action',
+            fn: actions.resetObjectMetadata(),
+            expectedActions: [dispatchAction.RESET_OBJECT_METADATA_ACTION()],
+        },
+        {
+            it: 'should return GET_OBJECT_METADATA_SUCCESS action -> test without prefix parameter',
```

Comment:
Can we provide a non-empty tags array? (type `TagSet`)

Reply:
done